glitch/apps.py

```diff
@@ -16,30 +16,52 @@
 )
 
 
-def glitch_image(input_path: str, output_path: str) -> None:
+def glitch_image(input_path: str, output_path: str, options: dict) -> None:
```

Comment:
A more pythoninc way to pass this options (specially if some of them have default value) will be using `kwargs`:
```python
def glitch_image(input_path: str, output_path: str, **kwargs: Any) -> None:
  block_movement = kwargs.get('block_movement', default_block movement)
  ...
```
or even more explicit:
```python
def glitch_image(input_path: str, output_path: str, block_movement=15, block_size=None) -> None:
  block_size = block_size or default_block_size # 2 ways of handling default values, should never have mutable types as default values in function signatures since they are memoized
```

in both cases, the call to this method will look like:
```python
glitch_image(filepath, glitched_filepath, **options)
```

Reply:
also, im not sure we need to handle the type casting here, the view should deal with this